README.md

```diff
@@ -77,6 +77,15 @@ Run the application locally (requires [docker-compose](https://docs.docker.com/c
 make run
 ```
 
+Alternatively, you could also run the application with
+[podman-compose](https://github.com/containers/podman-compose) by setting the
+`COMPOSE_ENGINE` variable (for now SELinux must be set to the **Permissive**
```

Comment:
Can we add a warning to indicate this is dangerous?

Reply:
Done. Also added it to the Makefile